src/platform/forms-system/src/js/utilities/ui/index.js

```diff
@@ -18,6 +17,19 @@ export function focusElement(selectorOrElement, options) {
   }
 }
 
+// Set focus on target _after_ the content has been updated
+export function focusOnChange(name, target = '.edit-btn', testDom) {
+  setTimeout(() => {
+    const selector = `[name="${name}ScrollElement"]`;
+    const el = (testDom || document).querySelector(selector);
+    // nextElementSibling = page form
+    const focusTarget = el?.nextElementSibling?.querySelector(target);
+    if (focusTarget) {
+      focusElement(focusTarget);
```

Comment:
I'm hesitant to mention this, but what are your thoughts on removing the `if`? It looks like `focusElement` is already doing a null check so this one doesn't _strictly_ seem necessary, but I wonder if not having it would be a bad practice.

Reply:
Yeah, I could remove it... I was just thinking that we didn't need to call the function if it was undefined, but either way works.

Reply:
Removed the `if` wrapper.